xbmc/interfaces/legacy/Window.cpp

```diff
@@ -687,6 +677,14 @@ namespace XBMCAddon
               DelayedCallGuard dcguard(languageHook);
               stillWaiting = WaitForActionEvent(100) ? false : true;
             }
+
+            // If application has quit, close the window
+            if (bModal && g_application.m_bStop)
+            {
+              CLog::Log(LOGDEBUG, "PYTHON: Application quit inside doModal(), closing window");
```

Comment:
Maybe log more info regarding which window is actually being closed?

Reply:
This info will appear earlier in the log, as the only window this stanza closes is the most recently active one. Besides, I don't see any member variables that would let us identify the window in the log, do you?

Reply:
Hum I think you may be right